pkg/influx/api.go

```diff
@@ -17,9 +20,11 @@ type API struct {
 	recorder Recorder
 }
 
-func (a *API) Register(router *mux.Router) {
+func (a *API) Register(router *mux.Router, authMiddleware middleware.Interface) {
 	registerer := route.NewMuxRegisterer(router)
-	registerer.RegisterRoute("/api/v1/push/influx/write", http.HandlerFunc(a.handleSeriesPush), http.MethodPost)
+
+	registerer.RegisterRoute("/api/v1/push/influx/write", authMiddleware.Wrap(http.HandlerFunc(a.handleSeriesPush)), http.MethodPost)
```

Comment:
please add a comment:  // org-specific routes (write/) require an org ID, whereas generic routes are readable by any client even without that header